app/resources/api/v1/npq_profile_resource.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class NpqProfileResource < JSONAPI::Resource
```

Comment:
How do these resources work?

Does jsonapi know to use this one on `NpqProfilesController` by use of clever naming convention?

Reply:
it seems to have default look up paths where it goes to look in a certain order. so this looks to be done through convention. I don't see an option to specify this by hand.

Reply:
Scary stuff. Alright, well, I guess it matches Rails so it is not too bad. Thanks for digging into it.